deep-learning/_posts/2018-06-27-Intractivesegmentation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: review
+title:  "Interactive Image Segmentation with Latent Diversity"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    http://vladlen.info/papers/interactive-segmentation.pdf
+cite:
+authors: "Zhuwen Li, Qifeng Chen, and Vladlen Koltun"
+title:   "Interactive Image Segmentation with Latent Diversity"
+venue:   "CVPR2018"
+---
+
+### Description
+
+What is interactive segmentation?
+
+It allows the user to segment the desired region in the image by successively placing positive
+(‘foreground’) and negative (‘background’) clicks. Every time a click is placed, the segmentation is updated. Once the segmentation matches the region intended by the user, the process concludes.
+
+In this paper, they trained a model which couples two convolution network. This model takes an image and the clicks as input and produces the segmentation on click object.
+
+First, a network synthesizes M segmentations f(x) = y, which y is a collection of segmentation masks. Second, a network g selects one of the M solutions synthesized by f. This intermediate representation y allows the network f to produce multiple clean segmentations and the network f produces
```

Comment:
This intermediate representation y allows the network f to....

Split that sentence in 2